addons/mail/populate/discuss/mail_message.py

```diff
@@ -28,4 +40,11 @@ def _populate(self, size):
                         "res_id": channel.id,
                     }
                 )
-        return res + self.env["mail.message"].create(messages)
+        res = self.env["mail.message"]
```

Comment:
The assignment to `res` above is not used ( `res = super()._populate(size)` ), but is it important to reassign `res` to an empty message recordset are there undesirable records in the one from super?